### ISSUE TITLE
Custom ids limitation

### ISSUE THREAD

**Issue by JulienPeloton** [astrolabsoftware/grafink#8]
In the [documentation](https://github.com/astrolabsoftware/grafink/blob/master/docs/LoadAlgorithm.md#disadvantages-1), the custom id are limited by

```scala
vertexCountBound = (1L << (TOTAL_BITS - partitionBits - USERVERTEX_PADDING_BITWIDTH));
// here TOTAL_BITS = 64, partitionBits = 16, USERVERTEX_PADDING_BITWIDTH = 3 for Normal Vertex
// So this evaluates to 17592186044416 ~ 17.5 trillion
```

But if I evaluate

```scala
1L << (64 - 16 - 3)
res0: Long = 35184372088832
```

which is twice the number quoted above. What do I miss here?

**Comment by saucam** [comment]
Nothing, this calculation is correct. I am not sure how I get half the number in the first place! Already fixed in PR #7

**Comment by JulienPeloton** [comment]
thanks!
